src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -34,15 +35,26 @@ import scala.util.{ Failure, Success, Try }
 class DepositServlet(app: EasyDepositApiApp)
   extends ProtectedServlet(app)
     with FileUploadSupport {
-  configureMultipartHandling(MultipartConfig())
+
+  configureMultipartHandling(app.multipartConfig.copy(location = app.multipartConfig.location
```

Comment:
I would much rather have this validation for `location` in `EasyDepositApiApp` as you read the properties from the configuration file. I think you can even reuse some code there for the validation...

Reply:
The submitter also does its own validation. In my opinion the app should be agnostic about the details of the components (in this case turn a relative path to an absolute path), just provide the values.